### PR TITLE
Fixed panels on why choose uk page

### PR DESCRIPTION
**Pull request by jessgilbert**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

image

### CODE REVIEW

File: great_international/panels/great_international.py [hunk 1]
```diff
@@ -1267,21 +1267,20 @@ class AboutUkWhyChooseTheUkPagePanels:
...
-                    MultiFieldPanel([
-                        FieldPanel('ebook_section_title'),
-                        FieldPanel('ebook_section_body'),
-                        HelpPanel('CTAs require both text and a link to show '
-                                  'on page. '),
-                        FieldPanel('ebook_section_cta_text'),
-                        FieldPanel('ebook_section_cta_link'),
-                    ])
-                ])
+                ]),
```
Comment (Kenneth-Siu) [inline great_international/panels/great_international.py]: Something like this....?

```
FieldRowPanel([
    ImageChooserPanel('blah'),
    MultiFieldPanel([
        HelpPanel('blah'),
        FieldPanel('blah'),
    ]),
])

Comment (jessgilbert) [reply great_international/panels/great_international.py]: It goes very very strange like that and I don't know why? :/ 
image

Comment (jessgilbert) [reply great_international/panels/great_international.py]: I think Help panel has a lot of spacing around, could I change it to be help text on `ebook_section_image_alt_text`?

Comment (jessgilbert) [reply great_international/panels/great_international.py]: Hows this? 
image

Comment (Kenneth-Siu) [reply great_international/panels/great_international.py]: Sick. 👌

**Pull request merged**